build/Publish.targets

```diff
@@ -33,11 +35,13 @@
                      Condition=" '$(IslinuxPortable)' != 'true' "/>
       <ForPublishing Include="%(GenerateArchivesInputsOutputs.Outputs)"
                      Condition=" '$(IsLinuxDistroSpecific)' != 'true' "/>
-      <ForPublishing Include="$(PackagesDirectory)/Microsoft*.nupkg"
```

Comment:
Is there a reason that this search string changed to Microsoft.DotNet.Cli.Utils.*.nupkg?

Reply:
To make it more explicit - or more exacting - less chance of the RegExp 'net' capturing an incorrect package.